src/main/java/com/github/princesslana/greedorama/commands/TransactionCommand.java

```diff
@@ -68,4 +68,36 @@ public DiscordResponse buy(Options options) {
             })
         .orElse(Format.error("Could not find price for " + symbol));
   }
+
+  @CommandHandler(commandName = "sell")
+  public DiscordResponse sell(Options options) {
+    if (options.number <= 0) {
+      return Format.error("Number of shares must be greater than zero");
+    }
+    if (request.getArgs().size() != 1) {
+      return Format.error("There should be exactly one stock symbol, but there isn't");
+    }
+
+    var symbol = request.getArgs().get(0);
+    var guildId = request.getDispatcher().guildFromEvent(request.getEvent());
+    var userId = request.getDispatcher().identityFromEvent(request.getEvent());
+    var user = users.get(guildId, userId);
+
+    return stocks
+        .get(symbol)
+        .map(
+            s -> {
+              var txn = Transaction.buy(user, s, -options.number);
```

Comment:
Let's make a `Transaction.sell` method

Reply:
Since they seem to be the exact same logic, can the `Transaction.buy` & `Transaction.sell` methods be generalized into a single `Transaction.create` method or something similar?

Reply:
I like the explicitness of buy and sell, but if overall it's cleaner to just have a create method then we can do that